animations/spells/1st/inside-ropes.json

```diff
@@ -0,0 +1,73 @@
+{
+	"spell:inside-ropes": [
+		{
+			"preset": "onToken",
+			"trigger": [
+				"effect",
+				"toggle",
+				"action"
+			],
+			"options": {
+				"scaleToObject": {
+					"value": 1
+				}
+			},
+			"contents": [
+				{
+					"predicate": [
+						"jb2a:patreon"
+					],
+					"file": "jb2a.vine.complete.void.single.03.dark_pinkpurple.2"
+				},
+				{
+					"predicate": [
+						"jb2a:free"
+					],
+					"file": "jb2a.vine.complete.nature.single.03.green.2"
+				}
+			]
+		}
+	],
+	"effect:inside-ropes-3rd-rank": [
+		{
+			"preset": "onToken",
+			"trigger": "effect",
+			"options": {
+				"scaleToObject": {
+					"value": 0.75
+				}
+			},
+			"contents": [
```

Comment:
There is no non-persistent version

Reply:
Well, the 3rd level version doesnt really need a non-persistent version as it will trigger on the original cast. It is there just to be creepy. I do not think we want to double the animation:
1. Casting
1. Attaching effect on token.

Reply:
I see you also added a `toggle` trigger, it really should not be the case. Also, if a spell has an effect, don't use `action`. Its already a given that if an effect exists, the spell "happens" when you add the effect.

Reply:
This is a unique spell that has an effect only when heightened. So I think toggle to get the initial graphic is fine as that is what most will use. If you heighten it you can draf the effect on the token and have it persistent.

Reply:
Toggle is triggered every time there is an update to the actor. It should be used sparingly. What you described is a cause for using only `action`.

Reply:
I don't even think `toggle` triggers in this usecase. In the future, check these things ahead of time with Animations History.

Reply:
I tested it with animation history, also tested every workflow I have done. Always test things before commiting them is my mantra. Even managed to get bless auras to expand but did not like the animation when it is 60 ft so decided to ignore it.